src/main/java/org/spongepowered/mod/SpongeMod.java

```diff
@@ -143,6 +141,8 @@ public void onPreInit(FMLPreInitializationEvent event) {
             SpongeForgeModuleRegistry.registerForgeData();
             SpongeModMessageHandler.init();
 
+            Class<?> clazz = Class.forName("org.spongepowered.api.entity.ai.task.AbstractAITask");
```

Comment:
Not really, but it causes the transformer to run earlier. If there's any sort of problem , it will show up earlier rather than later.